src/pages/add/csv.vue

```diff
@@ -0,0 +1,304 @@
+<template>
+  <PageHeader>
+    <template #left-btn>
+      <IconButton
+        @click="$router.push('/add')"
+        size="large"
+        color="normal"
+        icon-name="arrow_back"
+      ></IconButton>
+    </template>
+    <template #title>CSVファイルから追加</template>
+  </PageHeader>
+  <div class="main">
+    <div class="main__csv csv">
+      <p class="csv__header">CSVファイル</p>
+      <div v-if="fileName" class="csv__file">{{ fileName }}</div>
+      <div v-else class="csv__file--unselected">未選択</div>
+      <p class="csv__btn">
+        <Button @click="upload" size="small">アップロードする</Button>
+      </p>
+    </div>
+    <div class="main__courses courses">
+      <div class="courses__contents">
+        <template v-if="valid">
+          <CardCourse
+            v-for="{ checked, course } in courseData"
+            :key="course"
```

Comment:
一意の文字列に。
idって必ずあると思うので
```suggestion
            :key="course.id"
```